validator/validator.go

```diff
@@ -0,0 +1,14 @@
+package validator
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// Rule is the interface which is implemented
+// by every rule defined in validator package
+type Rule interface {
+	// Validate validates the cobra command
```

Comment:
Add name

Reply:
like different func name for each rule?

Reply:
Yes.. We need way to join text representation of rule with it's implementation. If that is not part of interface then there is no easy way to provide rule name as rules will be hardcoded inside lib.